app/src/main/java/arjun/hanabiandroid/game/InitialState.java

```diff
@@ -0,0 +1,38 @@
+package arjun.hanabiandroid.game;
+
+import android.support.annotation.IntDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+public class InitialState {
+    @Retention(RetentionPolicy.SOURCE)
+    @IntDef({
+            GAME_TYPE_NO_RAINBOW,
+            GAME_TYPE_RAINBOW_SUIT,
+            GAME_TYPE_RAINBOW_WILDCARD,
+    })
+    @interface GameType {
+    }
+
+    public static final int GAME_TYPE_NO_RAINBOW = 0;
+    public static final int GAME_TYPE_RAINBOW_SUIT = 1;
+    public static final int GAME_TYPE_RAINBOW_WILDCARD = 2;
+
+    @GameType
+    final int gameType;
+    final long seed;
+    final int numPlayers;
+    final int turnOrder;
```

Comment:
This choice makes sense if this is the sequence of events in the app:
- server determines player order (TBD)
- server beams down `InitialState` to all players (`N` of them), changing the turnOrder value accordingly
- each client constructs the first GameState
- after any turn is made and communicated to everyone (client -> server -> all clients ?),  then each client can check `bool isMyTurn=(state.moves.size() % numPlayers)==turnOrder` because no passing is allowed

is that what you had in mind? if so this looks good to me